bedrock/products/templates/products/vpn/windows-download.html

```diff
@@ -0,0 +1,110 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "products/vpn/download.html" %}
+
+{% block canonical_urls %}<meta name="robots" content="noindex,nofollow">{% endblock %}
+
+{% block page_title_full %}{{ ftl('vpn-windows-download-page-title') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('mozilla-vpn-platform-download') }}
+{% endblock %}
+
+{% block content %}
+  <div class="vpn-platform-download">
+    {% if block_download %}
+      <header class="platform-download-header">
+        <div class="mzp-l-content platform-download-blocked">
+        <img src="{{ static('img/products/vpn/download/vpn-unavailable-country.svg') }}" alt="">
+        <h1>{{ ftl('vpn-download-unable-to-download') }}</h1>
+        <p>{{ ftl('vpn-download-not-in-country') }}</p>
+        </div>
+      </header>
+    {% endif %}
+
+    {% if not block_download %}
+    <header class="platform-download-header">
+      <div class="mzp-l-content platform-download-header-wrapper">
+        <div class="vpn-logo">
+          <img src="{{ static('img/logos/vpn/vpn-dark-logo.svg') }}" alt="">
+        </div>
+        <h1>{{ ftl('vpn-windows-downloading-mozilla-vpn') }}</h1>
+        <p>{{ ftl('vpn-your-download-should-start', url=windows_download_url, id='vpn-download-link-win') }}</p>
+      </div>
+    </header>
+    <section class="vpn-download-instructions windows mzp-l-content">
+      <ol class="mzp-l-columns mzp-t-columns-four">
+        {% call picto(
+          base_el='li',
+          title=ftl('vpn-open-the-file'),
+          image=resp_img(
+            url='img/products/vpn/download/vpn-file-windows.png',
```

Comment:
Suggestion: it would be nice to add a responsive image for these so we're not serving larger images than are needed?

https://bedrock.readthedocs.io/en/latest/coding.html#resp-img